Polyhedron/include/CGAL/draw_polyhedron.h

```diff
@@ -46,8 +46,8 @@ void draw(const CGAL_POLY_TYPE& apoly,
   if (!cgal_test_suite)
   {
     CGAL::Qt::init_ogl_context(4,3);
-    int argc=2;
-    const char* argv[2]={"polyhedron_viewer","--old"};
+    int argc=1;
+    const char* argv[2]={"polyhedron_viewer","\0"};
```

Comment:
```suggestion
    const char* argv[2]={"polyhedron_viewer",""};
```

`""` is already a (null-terminated) string literal. See https://en.cppreference.com/w/cpp/language/string_literal.

Reply:
What is more: if `argc` is 1, then `argv` should be an array of size 1, and not 2.
```suggestion
    const char* argv[1]={"polyhedron_viewer"};
```

Reply:
Thanks @lrineau ; changed in all viewers cf. 
ee11c85 (I don't remember where this code comes from).

(BTW it is really nice to be able to run an example without needing to add a link to the data directory ! Good job @sloriot )